content.md

```diff
@@ -143,14 +143,14 @@
 
 ### 5.1 Text field
 
-Tap on form field and complete field with the pop-up keyboard.
+Tap on form field and and type the desired value using the the pop-up keyboard.
 
 ![screenshot](./images/5/text.jpg)
 <div class="pagebreak"></div>
 
-### 5.2 AutoLookUp - Text field
+### 5.2 Auto LookUp
 
-Start completing the form field and select the applicable AutoLookUp item from the list displayed below the text field
+Start completing the form field and select the applicable Auto LookUp item from the list displayed below the text field.
```

Comment:
'Auto Lookup' can be completely removed since it not a specific type of item. It is just a regular item in the list, the way the Auto Lookup field works is unique, but not the items that it lists.